packages/backend/src/main.ts

```diff
@@ -7,8 +7,16 @@ import AppModule from './modules/app.module';
 import Header from './common/enum/headers.enum';
 import { LoggerService } from './common/logger';
 import ValidationExceptionFilter from './common/error-handling/exception-filter';
+import { database, status } from 'migrate-mongo';
 
 const bootstrap = async (): Promise<void> => {
+  const { db } = await database.connect();
+  const migrations = await status(db);
+  if (!migrations?.length || migrations.pop()?.appliedAt === 'PENDING') {
+    console.error("Migrations are not applied, please run 'yarn migrate'");
```

Comment:
There are pending database migrations, please backup your database and run 'yarn migrate' to apply them.